### PR TITLE
Extract files github.py, redirects.py, settings.py

### PR DESCRIPTION
**Pull request by simon04**
Extract files github.py, redirects.py, settings.py to reduce size of main.py

To review consider the command `git diff main --color-moved`

Thanks!

### CODE REVIEW

File: versionhandler.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 import re
 from typing import List, Optional, Tuple
 
-logger = logging.getLogger(__name__)
+logger = logging.getLogger("github-wikidata-bot")
```
Comment (waldyrious) [inline versionhandler.py:5]: Just curious, what's this change about?

Comment (konstin) [reply versionhandler.py:5]: yeah that should remain `logger = logging.getLogger(__name__)`

**Review by konstin — APPROVED** [review]
:+1:

File: versionhandler.py (hunk 1 above)
Comment (simon04) [reply versionhandler.py:5]: We init the logger using 

```py
"loggers": {"github-wikidata-bot": {"handlers": handlers, "level": "INFO"}}
```

The logger name "github-wikidata-bot" must be used for every `logging.getLogger` call in order to yield the desired configuration.

`__name__` differs for every file. Reference: https://docs.python.org/3.11/library/__main__.html#name-main

Comment (konstin) [reply versionhandler.py:5]: normally you define the logger config for your whole package/module. Could you mode the files into a `github_wikidata_bot` module and change the logger config to snake case?

Comment (simon04) [reply versionhandler.py:5]: Done.

File: github_wikidata_bot/__main__.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env python3
+
+from .main import main
+
+if __name__ == "__main__":
+    main()
```
Comment (konstin) [inline github_wikidata_bot/__main__.py:6]: :+1:

**Comment by konstin** [comment]
Thank you!

**Pull request merged**